go/v1beta1/storage/pgsqlstore.go

```diff
@@ -0,0 +1,555 @@
+// Copyright 2017 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/fernet/fernet-go"
+	"github.com/golang/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/grafeas/grafeas/go/name"
+	pb "github.com/grafeas/grafeas/proto/v1beta1/grafeas_go_proto"
+	prpb "github.com/grafeas/grafeas/proto/v1beta1/project_go_proto"
+	"github.com/lib/pq"
+	"golang.org/x/net/context"
+	fieldmaskpb "google.golang.org/genproto/protobuf/field_mask"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type pgSQLStore struct {
+	*sql.DB
+	paginationKey string
+}
+
+func NewPgSQLStore(config *PgSQLConfig) *pgSQLStore {
+	err := createDatabase(createSourceString(config.User, config.Password, config.Host, "postgres", config.SSLMode), config.DbName)
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+	db, err := sql.Open("postgres", createSourceString(config.User, config.Password, config.Host, config.DbName, config.SSLMode))
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+	if db.Ping() != nil {
+		log.Fatal("Database server is not alive")
+	}
+	_, err = db.Exec(createTables)
+	if err != nil {
+		db.Close()
+		log.Fatal(err.Error())
```

Comment:
You can do 
if _, err := db.Exec(); err != nil 
here

Reply:
good point, done